recipes/miktex-portable/bld.bat

```diff
@@ -0,0 +1,16 @@
+
+
+7za x miktex-portable-%PKG_VERSION%.exe -o%LIBRARY_PREFIX%\miktex-portable
+if errorlevel 1 exit 1
+
+
+rem SCRIPTS dir should already be created by 7za install
+mkdir "%SCRIPTS%"
+if errorlevel 1 exit 1
+
+rem latex tools must be run from miktex tree
+for %%f in ("%LIBRARY_PREFIX%\miktex-portable\miktex\bin\*.exe") do (
+	echo @%%~dp0\..\Library\miktex-portable\miktex\bin\%%~nf %%* >> "%SCRIPTS%\%%~nf.bat"
```

Comment:
You're right on point there. Conda would rewrite it automatically if you relied on the env vars provided by conda-build such as `%library_prefix%`.

I feel like that is actually a little bit more resistant to future changes than the current approach which embeds some knowledge of the directory structure into the script that could be abstracted away into the env vars as explained.